### PR TITLE
Add foreman nightly config file with koji tag info

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
@xprazak2 Here is the nightly config

@jlsherrill mind vetting this given its the first time we've introduced Foreman tag management here?

**Comment by xprazak2** [comment]
When running a dry run I am getting:
```
[vagrant@centos-packaging tool_belt]$ ./tools.rb koji configs/foreman_nightly.yaml


PROCESSING foreman-nightly-rhel7
[noop] kkoji regen-repo foreman-nightly-rhel7-build


PROCESSING foreman-nightly-nonscl-rhel7
[noop] kkoji regen-repo foreman-nightly-nonscl-rhel7-build


PROCESSING foreman-plugins-nightly-rhel7
[noop] kkoji remove-external-repo rhel-7.0-server-optional foreman-plugins-nightly-rhel7-build
[noop] kkoji regen-repo foreman-plugins-nightly-rhel7-build


PROCESSING foreman-plugins-nightly-nonscl-rhel7
[noop] kkoji remove-external-repo rhel-7.0-server-optional foreman-plugins-nightly-nonscl-rhel7-build
[noop] kkoji remove-tag-inheritance foreman-plugins-nightly-nonscl-rhel7-build foreman-plugins-nightly-nonscl-rhel7-override
[noop] kkoji remove-tag-inheritance foreman-plugins-nightly-nonscl-rhel7-build foreman-nightly-nonscl-rhel7
[noop] kkoji remove-tag-inheritance foreman-plugins-nightly-nonscl-rhel7-build foreman-nightly-rhel7
[noop] kkoji add-tag-inheritance --priority=0 foreman-plugins-nightly-nonscl-rhel7-build foreman-plugins-nightly-nonscl-rhel7-override
[noop] kkoji add-tag-inheritance --priority=10 foreman-plugins-nightly-nonscl-rhel7-build foreman-nightly-rhel7
[noop] kkoji add-tag-inheritance --priority=15 foreman-plugins-nightly-nonscl-rhel7-build foreman-nightly-nonscl-rhel7
[noop] kkoji regen-repo foreman-plugins-nightly-nonscl-rhel7-build


PROCESSING foreman-nightly-rhel7-dist


PROCESSING foreman-plugins-nightly-rhel7-dist
```

but I cannot tell if it is expected or not.

File: configs/foreman_nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      foreman-plugins-nightly-nonscl-rhel7-override: null
+    build_target: foreman-plugins-nightly-nonscl-rhel7-build
+    build_package_group_source_tag: foreman-plugins-nightly-nonscl-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      foreman-plugins-nightly-nonscl-rhel7-build:
+        foreman-plugins-nightly-nonscl-rhel7-override: 0
+        foreman-nightly-rhel7: 10
+        foreman-nightly-nonscl-rhel7: 15
```
Comment (jlsherrill) [inline configs/foreman_nightly.yaml]: Looks like the 10 and 15 are switched (from taginfo):
```
Inheritance:
  0    .... foreman-plugins-nightly-nonscl-rhel7-override [17]
  10   .... foreman-nightly-nonscl-rhel7 [5]
  15   .... foreman-nightly-rhel7 [8]
```

**Comment by xprazak2** [comment]
Seems ok now:

```
[vagrant@centos-packaging tool_belt]$ ./tools.rb koji configs/foreman_nightly.yaml 


PROCESSING foreman-nightly-rhel7
[noop] kkoji regen-repo foreman-nightly-rhel7-build


PROCESSING foreman-nightly-nonscl-rhel7
[noop] kkoji regen-repo foreman-nightly-nonscl-rhel7-build


PROCESSING foreman-plugins-nightly-rhel7
[noop] kkoji regen-repo foreman-plugins-nightly-rhel7-build


PROCESSING foreman-plugins-nightly-nonscl-rhel7
[noop] kkoji regen-repo foreman-plugins-nightly-nonscl-rhel7-build


PROCESSING foreman-nightly-rhel7-dist


PROCESSING foreman-plugins-nightly-rhel7-dist
```

**Pull request merged**